docs/core/index.md

```diff
@@ -13,15 +13,15 @@ See [About .NET Core](about.md) to learn more about .NET Core, including its cha
 
 Check out [.NET Core Tutorials](tutorials/index.md) to learn how to create a simple .NET Core application. It only takes a few minutes to get your first app up and running. If you want to try .NET Core in your browser, look at the [Numbers in C#](../csharp/tutorials/intro-to-csharp/numbers-in-csharp.yml) online tutorial.
 
-## Download .NET Core 2.2
+## Download .NET Core
 
-Download the [.NET Core  2.2 SDK](https://www.microsoft.com/net/download) to try .NET Core on your Windows, macOS, or Linux machine. Visit [dotnet/core](https://hub.docker.com/_/microsoft-dotnet-core/) if you prefer to use Docker containers.
+Download the [.NET Core SDK](https://www.microsoft.com/net/download) to try .NET Core on your Windows, macOS, or Linux machine. Visit [dotnet/core](https://hub.docker.com/_/microsoft-dotnet-core/) if you prefer to use Docker containers.
```

Comment:
not related to your PR: should the visible text mention that these are docker images? I'd read "dotnet/core" as a NET thing, not the .NET page on DockerHub.

Reply:
Good point! Made some changes.